docs/limitations.md

```diff
@@ -1,12 +1,30 @@
 # Current limitations
 
-1. Does not work on Windows. Tested on Mac and Linux only. Tracking issue: https://github.com/prisma/prisma2/issues/4
-2. Processes 1 request at a time, no parallelism during the early preview period. This will be fixed before GA. 
-3. Some edge cases for complex nested mutations don't work properly. 
-4. Limited auto-complete in Typescript projects due to a compiler bug. We raised a PR to fix that [here](https://github.com/microsoft/TypeScript/pull/32100). 
-5. No realtime API/subscriptions. In the future, we will have Prisma events engine (after the GA) that would pave way for such a feature, there is no ETA for this yet. 
-6. Models must have an `@id` attribute and it must take one of these forms:
+During the preview phase of Prisma 2 there are missing features, limited performance and stability issues you should be aware of:
+
+## Functionality limitations
+
+- `Embed` types are not implemented yet ([tracking issue](https://github.com/prisma/lift/issues/43))
+- Many other types are not fully implemented yet
+- Models must have an `@id` attribute and it must take one of these forms:
     - `Int @id`
     - `String @id @default(uuid())`
-    - `String @id @default(cuid())`
-7. When [introspecting](./introspection.md) a database, Prisma only recognizes many-to-many relations that follow the Prisma conventions for [relation tables](https://github.com/prisma/prisma2/blob/master/docs/relations.md#mn).
+    - `String @id @default(cuid())`  
+- When [introspecting](./introspection.md) a database, Prisma for now only recognizes many-to-many relations that follow the Prisma conventions for [relation tables](https://github.com/prisma/prisma2/blob/master/docs/relations.md#mn).
+- Some edge cases for complex nested mutations don't work properly yet.
+- Non-interactive TTYs (like Git Bash on Windows) are currently not supported by Prisma2 CLI ([tracking issues](https://github.com/prisma/prisma2/issues/554))
```

Comment:
I had to look up what a TTY is 🙈

Reply:
How can we fix that? Also not happy with using that, but "non interactive tty" is a thing people say.

Reply:
maybe just call it "shells" instead? I'm still not sure what the exact meaning of TTY is here but is there a simpler term for this?

Reply:
```suggestion
- Non-interactive shells/TTYs (like Git Bash on Windows) are currently not supported by Prisma2 CLI ([tracking issues](https://github.com/prisma/prisma2/issues/554))
```

Reply:
Ping @pimeys, I assume you know a bit about that and might help us out if my suggestion is a valid one.

Reply:
TTY = Terminal Type.

Reply:
It's a very common term when talking about computers.

Reply:
Thanks.
Is it valid to do "shells/TTYs"?

Reply:
Shell is a program such as bash or zsh. Terminal is a program such as the Windows terminal application, xterm, linux console et.al. emulating the original teletype. So shell is wrong here, and you can use an internactive terminal on Windows and with that, bash.

Reply:
@nikolasburk Your call on this then - I would just keep "TTY" and maybe provide an appropriate link that maybe explains what "non interactive tty" actually is or not.

Reply:
(or use the word terminal)

Reply:
```suggestion
- Non-interactive terminals (like Git Bash on Windows) are currently not supported by Prisma2 CLI ([tracking issues](https://github.com/prisma/prisma2/issues/554))
```

Reply:
That definitely works for me. Thanks @pimeys